swift/Sources/Xpring/main.swift

```diff
@@ -17,43 +17,23 @@ let payID = "alice$dev.payid.xpring.money"
 // The network to resolve on.
 let network = XRPLNetwork.test
 
-
-// Run items on a background thread because `XpringClient` always calls back on the main thread and there is no
-// synchronous API.
-// TODO(keefertaylor): Clean this up when XpringKit supports thread management or synchronous APIs in XpringClient.
-let dispatchGroup = DispatchGroup()
-
-DispatchQueue.global(qos: .background).async {
-
-  print("\nUsing rippled node located at: \(grpcURL)\n")
-  let xrpClient = XRPClient(grpcURL: grpcURL, network: network)
-
-  print("Using network: \(network)\n")
-  let payIDClient = XRPPayIDClient(xrplNetwork: network)
-
-  let xpringClient = try! XpringClient(payIDClient: payIDClient, xrpClient: xrpClient)
-
-  print("Sending:");
-  print("- Drops \(dropsToSend)")
-  print("- To: \(payID)")
-  print("- From: \(wallet.address)\n")
-
-  dispatchGroup.enter()
```

Comment:
What happened to this `dispatchGroup` stuff?

Reply:
We actually just built synchronous APIs that work better for the demos.